src/server/ua_subscription_events.c

```diff
@@ -234,7 +319,22 @@ UA_Server_evaluateWhereClauseContentFilter(UA_Server *server,
             return UA_STATUSCODE_BADEVENTFILTERINVALID;
         }
         case UA_FILTEROPERATOR_EQUALS:
-        case UA_FILTEROPERATOR_ISNULL:
+        case UA_FILTEROPERATOR_ISNULL: {
```

Comment:
Looks good!
But please break out the cases into dedicated functions.
Those are much easier to reason about as the number of input arguments is small.
If that function returns a statuscode, we can have only one line per case.